receiver/mysqlreceiver/metadata.yaml

```diff
@@ -279,3 +283,13 @@ metrics:
       monotonic: false
       aggregation: cumulative
     attributes: [threads]
+  mysql.connection_errors:
```

Comment:
It seems like a `mysql.connection` namespace could be useful for other metrics such as `.count`, which would be related to this metric. What do you think about calling this `mysql.connection.errors`?

Reply:
sure, done